src/utils/starknetId.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-param-reassign */
+import BN from 'bn.js';
+
+import { StarknetChainId } from '../constants';
+
+const basicAlphabet = 'abcdefghijklmnopqrstuvwxyz0123456789-';
+const basicSizePlusOne = new BN(basicAlphabet.length + 1);
+const bigAlphabet = '这来';
+const basicAlphabetSize = new BN(basicAlphabet.length);
+const bigAlphabetSize = new BN(bigAlphabet.length);
+const bigAlphabetSizePlusOne = new BN(bigAlphabet.length + 1);
+
+export function useDecoded(encoded: BN[]): string {
```

Comment:
can we get a test for this too?

Reply:
so we can catch bugs early and not wait for the integration tests (with network requests) to fail

Reply:
Ok no problem